src/dcap_provider.cpp

```diff
@@ -310,10 +322,16 @@ sgx_plat_error_t get_unescape_header(
 
     result = get_raw_header(curl, header_item, &raw_header);
     if (result != SGX_PLAT_ERROR_OK)
+    {
+        log(SGX_QL_LOG_INFO,
+            "Failed to escape header %s",

```

Comment:
"Failed to escape header"? Isn't this function get_unescape_header?

Reply:
Thanks fixed